msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.ObjCBinding.Common.After.targets

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@emaf / @mauroa 

I ran into one issue when testing this locally for legacy Xamarin: when I unzipped msbuild.zip into `C:\Program Files\Microsoft Visual Studio\2022\<Channel>\MSBuild\Xamarin`, this file wasn't removed and caused problems in the build (task `GetPropertyValue` does not exist). I fixed the problem by removing this file manually, but would this cause problems when upgrading Xamarin.iOS on Windows? Should I instead just remove the `_SetRemotePropertyValues` target and keep the (empty) file (so that we overwrite existing files on upgrades)?

Reply:
Oh, that's a good question, but I think VS will replace the entire folder in an update scenario, it won't just replace files, so deleting the file should be safe.